src/components/Search/Search.js

```diff
@@ -0,0 +1,12 @@
+import React, { useState } from 'react';
+
+export default function Search({ setQuery }) {
+  const [input, setInput] = useState('');
+
+  return (
+    <div>
+      <input onChange={(e) => setInput(e.target.value)} placeholder="Search" />
+      <button onClick={() => setQuery(input)}>Search</button>
+    </div>
```

Comment:
Great way to prevent multiple fetch calls to the API by setting two different values of the same state with an onClick function.